### ISSUE TITLE
File d-leveldb bugs

### ISSUE THREAD

**Issue by truedat101** [IoTone/TheShellProject#15]
WARNING: A deprecated branch based version specification is used for the dependency d-leveldb. Please use numbered versions instead. Also note that you can still use the dub.selections.json file to override a certain dependency to use a branch instead.
Performing "debug" build using ldc2 for x86_64.
d-leveldb ~master: building configuration "library"...
..\..\..\..\..\AppData\Roaming\dub\packages\d-leveldb-master\d-leveldb\src\leveldb\db.d(32,16): Deprecation: module std.c.string is deprecated - Import core.stdc.string instead
shell-d ~master: building configuration "application"...
Copying files for leveldb...

**Comment by truedat101** [comment]
Wow, it's worse, the dub code package has a bad example, and the code segfaults

**Comment by pedroalvesbatista** [comment]
Right, we can send the pull require to take a good understanding in a full level, this bugs can show a lot about the levedb core

**Comment by truedat101** [comment]
The thing that concerns me about this library (dleveldb) and the demios leveldb is they are old, almost 4 years old with zero maintenance.  This drives me crazy about D.
